src/NUnitTestAdapter/TestConverter.cs

```diff
@@ -159,14 +159,24 @@ public DiaNavigationData GetNavigationData(string className, string methodName)
         {
             if (this.DiaSession == null) return null;
 
+            // First try using the class and method names provided directly
             var navData = DiaSession.GetNavigationData(className, methodName);
```

Comment:
I decided not to make this efficiency improvement. If we end up replacing the use of DiaSession with Mono.Cecil, no AppDomain will be needed. if we don't we can come back to this.